### PR TITLE
li42 recipe: add li42 results; fix bug in adding language id "zh_TW"

### PR DESCRIPTION
**Pull request by houwenxin**

### CODE REVIEW

File: egs/li42/asr1/run.sh [hunk 1]
```diff
@@ -56,7 +56,103 @@ set -o pipefail
 
 train_set=train_li42
 train_dev=dev_li42
-recog_set="dt_de_voxforge dt_en_wsj dt_es_voxforge dt_fr_voxforge dt_it_voxforge dt_ja_csj dt_nl_voxforge dt_pt_voxforge dt_ru_voxforge dt_zh_hkust et_de_voxforge et_en_wsj et_es_voxforge et_fr_voxforge et_it_voxforge et_ja_1_csj et_ja_2_csj et_ja_3_csj et_nl_voxforge et_pt_voxforge et_ru_voxforge et_zh_hkust dt_ca_commonvoice dt_fa_commonvoice dt_zh_TW_commonvoice et_eu_commonvoice et_tt_commonvoice dt_cy_commonvoice dt_fr_commonvoice et_ca_commonvoice et_fa_commonvoice et_zh_TW_commonvoice dt_de_commonvoice dt_it_commonvoice et_cy_commonvoice et_fr_commonvoice dt_en_commonvoice dt_kab_commonvoice et_de_commonvoice et_it_commonvoice dt_es_commonvoice dt_ru_commonvoice et_en_commonvoice et_kab_commonvoice dt_eu_commonvoice dt_tt_commonvoice et_es_commonvoice et_ru_commonvoice"
+# Full reognition set
+recog_set="
+	dt_amharic_babel et_amharic_babel 
+
```
Comment (sw005320) [inline egs/li42/asr1/run.sh:62]: Can you remove blank lines?
Also, you could probably use `{dt,et}_*_babel` to shorten the list.
The list is too long

Comment (houwenxin) [reply egs/li42/asr1/run.sh]: Thank you for your advice. Results are uploaded. I use regular expression for the recog set to shorten the list.

Comment (houwenxin) [reply egs/li42/asr1/run.sh]: Okay. I have uploaded RESULTS.md and updated the recipe.

**Comment by sw005320** [comment]
LGTM. 42 language result is quite impressive. Thanks a lot!

**Pull request merged**